osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -154,21 +157,21 @@ private void load(OsuConfigManager config, OsuColour colours, ChannelManager cha
                                     RelativeSizeAxes = Axes.Both,
                                     Colour = Color4.Black,
                                 },
-                                channelTabControl = new ChannelTabControl
+                                ChannelTabControl = CreateChannelTabControl().With(d =>
```

Comment:
this is kinda ugly.

Reply:
Yeah, but I don't know how else to expose the available tabs to the test (without changing the accessibility on framework side)